test/components/line_overlap.dart

```diff
@@ -0,0 +1,223 @@
+import 'dart:convert';
```

Comment:
from what I see, you already implemented everything for lineOverlap in here? No extra PR? BTW I'm fine with that 😄

Reply:
I am working on that in a separate branch, it had leaked to this one during stash/pop, I let it be.